content/docs/user-guide/how-to/resolve-merge-conflicts.md

```diff
@@ -36,7 +36,8 @@ stages:
 ## `dvc.lock`
 
 There's no need to resolve lock file merge conflicts manually. You can safely
-overwrite this file by using `dvc repro` after merging `dvc.yaml`.
+overwrite this file by using `dvc repro` after merging `dvc.yaml`. You may need
```

Comment:
Hmm, I'm not sure I understand the last two sentences together ... safely overwrite ... may need to remove first?

Reply:
Maybe it's better to say that you can safely drop `dvc.lock` and reproduce a clean one with `dvc repro`?

Reply:
yes, I think that would be fine